mozilla/bedrock
# [fix bug 1423436] Whatsnew page promoting Rocket for Indonesia

## Description
Adds a custom whatsnew page promoting Rocket, exclusively in Indonesia, and only for versions 57.0.* and up. All other locales/versions should get their version-appropriate localized page.

Text is hard-coded in Indonesian so no l10n needed (I didn't even wrap the strings). If/when Rocket is ever available in other countries we'll need to rethink this but for now I think hard-coding strings is the way to go.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1423436

## Testing
https://bedrock-demo-craigcook.us-west.moz.works/id/firefox/58.0/whatsnew/

- Verify the page is only shown when locale == id.
- Verify the page is only shown when version is 57.0.* or higher.
